packages/internals/src/cli/getSchema.ts

```diff
@@ -24,14 +24,18 @@ export type GetSchemaResult = {
  * Async
  */
 
-type ReturnWithNonNullArg<In, Out> = In extends undefined ? Out | null : Out
-
+export async function getSchemaPath(): Promise<GetSchemaResult | null>
```

Comment:
Just for my understanding: overaloads were added to exclude "empty schema path, non-empty options" situation, right?

Reply:
They were added so that the `Promise` result is only nullable when no first argument (the optional schema path to look in) is provided. This makes object destructuring more convenient